src/SFA.DAS.DATA.MANAGEMENT/SFA.DAS.Data.Management.Database.sqlproj

```diff
@@ -268,5 +268,24 @@
     <Build Include="Schemas\Comp.sql" />
     <Build Include="Views\Comp_DAS_Compliance_Report .sql" />
     <Build Include="Tables\Stg_GA_SessionDataDetail.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_NationalAchievementRate.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_NationalAchievementRateOverall.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_ProviderRegistration.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_ProviderRegistrationFeedbackAttribute.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_ProviderRegistrationFeedbackRating.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_ProviderStandard.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_ProviderStandardLocation.sql" />
+    <Build Include="Tables\ASDataPL_CRSDel_StandardLocation.sql" />
+    <Build Include="Tables\ASDataPL_Provider.sql" />
+    <Build Include="Tables\ASDataPL_CRS_ApprenticeshipFunding.sql" />
+    <Build Include="Tables\ASDataPL_CRS_FrameworkFundingPeriod.sql" />
+    <Build Include="Tables\ASDataPL_CRS_LarsStandard.sql" />
+    <Build Include="Tables\ASDataPL_CRS_SectorSubjectAreaTier2.sql" />
+    <Build Include="StoredProcedure\ImportProviderToPL.sql" />
+    <Build Include="Tables\ASDataPL_CRS_Framework.sql" />
+    <Build Include="Tables\ASDataPL_CRS_SectorStandard.sql" />
+    <Build Include="StoredProcedure\ImportCRSFrameworkToPL.sql" />
+    <Build Include="StoredProcedure\ImportCRSSectorStandardToPL.sql" />
+    <Build Include="StoredProcedure\BuildCRSPresentationLayer.sql" />
```

Comment:
Can we name them FAT2 instead of CRS Please?

Reply:
Changes are deployed to AT